cpp/ql/test/query-tests/Likely Bugs/Format/NonConstantFormat/test.cpp

```diff
@@ -163,3 +163,22 @@ void fmt_with_assignment() {
   x = y = "a";
   printf(y); // GOOD
 }
+
+int wprintf(const wchar_t *format,...);
+typedef wchar_t *STRSAFE_LPWSTR;
+typedef const wchar_t *STRSAFE_LPCWSTR;
+typedef unsigned int size_t;
+
+void StringCchPrintfW(
+  STRSAFE_LPWSTR  pszDest,
+  size_t          cchDest,
+  STRSAFE_LPCWSTR pszFormat,
+   ...             
+);
+
+void wchar_t_test_good(){
+  wchar_t wstr[100];
+  StringCchPrintfW(wstr, 100, L"STRING"); 
```

Comment:
Is this the false positive?  You need to accept the changes to the `.expected` file (even if they're not desirable - the `.expected` shows the results that are expected in practice right now).  Also add an annotation to this line along the lines of `// BAD [NOT DETECTED]` so that future people looking at this line can easily tell what's going on.

Reply:
It is a false positive.